templates/_partials/helpers.tpl

```diff
@@ -14,3 +14,128 @@
     >
   </a>
 {/function}
+
+{assign
+  var=iconsMap
+  value=[
+    "cart" => "shopping_cart"
+  ]
+  scope="global"
+}
+
+
+{** SVG SPRITE **}
+
+<svg class="d-none">
+  <defs>
+    <g id="shopping_cart">
+      <path
+        d="M280-80q-33 0-56.5-23.5T200-160q0-33 23.5-56.5T280-240q33 0 56.5 23.5T360-160q0 33-23.5 56.5T280-80Zm400 0q-33 0-56.5-23.5T600-160q0-33 23.5-56.5T680-240q33 0 56.5 23.5T760-160q0 33-23.5 56.5T680-80ZM208-800h590q23 0 35 20.5t1 41.5L692-482q-11 20-29.5 31T622-440H324l-44 80h480v80H280q-45 0-68-39.5t-2-78.5l54-98-144-304H40v-80h130l38 80Z"/>
+    </g>
+  </defs>
+</svg>
+
+
+{**
+ * @param string iconName
+ * @param array extraAttributes
+ * @return string
+ * @example
+ * {renderIcon name='cart' extraAttributes=['class' => 'my-extra-class']}
+ *}
+{function renderThemeIcon iconName="" extraAttributes=[]}
+    {if $iconName && !empty($iconsMap[$iconName])}
+
+      <span
+        class="material-icons {if !empty($extraAttributes['class'])}{$extraAttributes['class']}{/if}"
```

Comment:
Here we have `material-icons` hardcoded so what would happen if someone wanted to use another icon lib? They override this smarty template?

Reply:
@matks thank you for your response. 
Tbh modules aren't aware what icon fonts theme use. It's theme responsibility do display proper icon to given icon key. If you would like to change icon fonts to for example font-awsome, all you have to do is changing material icons to fa and map keys to your new icon-fonts.
Instead of:
```tpl
{assign
  var=iconsMap
  value=[
    "cart" => "shopping_cart"
  ]
  scope="global"
}
```

for example:

```tpl
{assign
  var=iconsMap
  value=[
    "cart" => "fa-shopping-cart" // not sure it's valid icon class
  ]
  scope="global"
}
```